site/src/theme/components/MuiIconButtonThemeOptions.ts

```diff
@@ -0,0 +1,38 @@
+import { Components } from "@mui/material";
+import { customColors } from "../palette";
+import { defaultTheme } from "../util";
```

Comment:
With the callback approach for setting styleOverrides, we don't need to import the defaultTheme anymore. 
See an example here => https://mui.com/blog/callback-support-in-style-overrides/

Reply:
Good spot, thank you Teeno! This removes the need for importing `defaultTheme` and `customColors` in most circumstances, which is great improvement. Have made this change here: https://github.com/blockprotocol/blockprotocol/pull/203/commits/1a271c7165a5922f3f98741d3f3d94e69b8d122b